lib/Dancer2/Core/App.pm

```diff
@@ -889,7 +889,11 @@ sub send_file {
                 [ $res->status, $res->headers_to_array, $fh ]
             );
         };
+
+        Scalar::Util::weaken( my $weak_self = $self );
+
         $response = Dancer2::Core::Response::Delayed->new(
+            error_cb => sub { $weak_self->logger_engine->log(@_) },
```

Comment:
Pinging again on this, @veryrusty, I think I tested it when committing it and it worked. Basically when you call log, it just passes it on to the logger. Are you sure this is a mistake?